operator/pkg/gateway-api/gamma_reconcile.go

```diff
@@ -55,7 +55,7 @@ func (r *gammaReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl
 	// Ignore deleting Gateway, this can happen when foregroundDeletion is enabled
```

Comment:
nit: :point_up: - would be nice if we could get a log message "Reconciling GAMMA Service" similar to the one we have for the gateway reconciler.

```
scopedLog.InfoContext(ctx, "Reconciling GAMMA Service")
```

Reply:
Nice idea, added.